website/pages/docs/extending/plugins.mdx

```diff
@@ -141,31 +149,80 @@ There are two steps involved in creating a plugin:
 2.  Serve the interface by calling the appropriate plugin serving method in
     your main method.
 
-A basic example is shown below. In this example, assume the `Builder` struct
-implements the `packer.Builder` interface:
+Basic examples are shown below. Note that if you can define a multi-plugin
+binary as it will allow to add more that one plugin per binary.
+
+
+<Tabs>
+<Tab heading="Multi-plugin Binary">
 
 ```go
 import (
   "github.com/hashicorp/packer/packer/plugin"
 )
 
 // Assume this implements packer.Builder
-type Builder struct{}
+type ExampleBuilder struct{}
+
+// Assume this implements packer.PostProcessor
+type FooPostProcessor struct{}
+
+// Assume this implements packer.Provisioner
+type BarProvisioner struct{}
 
 func main() {
- 	server, err := plugin.Server()
+	pps := plugin.NewSet()
+	pps.RegisterBuilder("example", new(ExampleBuilder))
+	pps.RegisterPostProcessor("foo", new(FooPostProcessor))
+	pps.RegisterProvisioner("bar", new(BarProvisioner))
+	err := pps.Run()
 	if err != nil {
-		panic(err)
+		fmt.Println(err.Error())
+		os.Exit(1)
 	}
-	server.RegisterBuilder(new(Builder))
-	server.Serve()
 }
 ```
 
-**That's it!** `plugin.Server` handles all the nitty gritty of
+**That's it!** `plugin.NewSet` handles all the nitty gritty of
 communicating with Packer core and serving your builder over RPC. It can't get
 much easier than that.
 
+Here the name of the plugin will be used to use each plugin, so if your plugin
+is named `my`, this would make the following parts available:
```

Comment:
```suggestion
is named `packer-plugin-my`, this would make the following parts available:
```
I think is clear if we give the full name of the plugin here

Reply:
Yup ! Thanks !